### PR TITLE
Add service account option

### PR DESCRIPTION
**Pull request by quizy101**
The chart does not support the use of service accounts. This feature is needed is some security contexts where API access to these resources may need to be limited.

I've set the default for `.Values.serviceAccount.create` to `false` to prevent any migration issues for other users. Also, it will use the service account `default` on new installs just like it does now when set to `false`.

### CODE REVIEW

File: privatebin/Chart.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: A Helm chart for installing PrivateBin
 name: privatebin
 home: https://privatebin.info/
 icon: https://raw.githubusercontent.com/PrivateBin/assets/master/images/preview/icon.png
-version: 0.5.2
+version: 0.5.3
```
Comment (bdashrad) [inline privatebin/Chart.yaml:8]:
```suggestion
version: 0.6.0
```

**Review by bdashrad — CHANGES_REQUESTED** [review]
looks good, but update the minor version number instead since we're adding features.

**Review by bdashrad — APPROVED** [review]

**Comment by jj0b** [comment]
@quizy101 

Since this was merged today ArgoCD is no longer able to sync privatebin in my cluster.

```
GROUP  KIND        NAMESPACE   NAME                STATUS     HEALTH   HOOK  MESSAGE
apps   Deployment  privatebin  privatebin          OutOfSync  Missing        error validating data: ValidationError(Deployment.spec): unknown field "serviceAccountName" in io.k8s.api.apps.v1.DeploymentSpec
```

Is there something I need to update to accommodate this change?

**Comment by quizy101** [comment]
@jj0b I have a fix at #30 hopefully these changes can get merged quickly. Sorry for the confusion.

**Comment by jj0b** [comment]
@quizy101 no worries. Thanks!

**Comment by jj0b** [comment]
@quizy101 confirmed working. :-) Cheers!

**Pull request merged**